app/services/typicode.js

```diff
@@ -0,0 +1,27 @@
+const request = require('request-promise'),
+  typicodePath = 'https://jsonplaceholder.typicode.com',
+  apiError = require('../errors');
+
+function options(endpoint) {
+  return {
+    uri: endpoint,
+    json: true,
+    resolveWithFullResponse: false
+  };
+}
+
+exports.requestAlbums = () => {
+  try {
+    return request(options(`${typicodePath}/albums`));
+  } catch (error) {
+    throw apiError(error.message);
+  }
```

Comment:
Why use try catch? The `request` function returns a promise, so you could simply do
`return request(options(${typicodePath}/albums)).catch((error)=> {throw apiError(error.message);})`. Saves you from having to write `try {...}`